Content/posts/common-templates-available.md

```diff
@@ -0,0 +1,17 @@
+---
+date: 2022-03-04 12:55
+description: For ease-of-use, we made some commonly used templates readily available
+tags: template
+author: Benny
+---
+# Commonly used templates available
+
+Today we've made a few commonly used Vapor templates readily available for you to use. These templates are automatically generated from the [Vapor's template](https://github.com/vapor/template) so that they'll always be up-to-date.  
```

Comment:
```suggestion
Today we've made a few commonly used Vapor templates readily available for you to use. These templates are automatically generated from [Vapor's main template](https://github.com/vapor/template) so that they'll always be up-to-date.  
```